nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Upsampling2d.java

```diff
@@ -56,6 +58,13 @@ public Upsampling2d(SameDiff sameDiff, SDVariable input, boolean nchw, int scale
     public Upsampling2d() {}
 
 
+    public Upsampling2d(INDArray input, int scale) {
+    }
+
+    public Upsampling2d(INDArray input, INDArray scaleH, INDArray scaleW, boolean nchw) {
+    }
```

Comment:
Constructors are not implemented.
Second one has wrong signature - should be
`Upsampling2d(INDArray input, int scaleH, int scaleW, boolean nchw)`

Compare to the existing constructors here (SameDiff one)